src/server/req_runjob.c

```diff
@@ -989,6 +989,17 @@ svr_strtjob2(job *pjob, struct batch_request *preq)
 	job_attr_def[(int) JOB_ATR_Comment].at_decode(
 			&pjob->ji_wattr[(int) JOB_ATR_Comment], (char *) 0,
 			(char *) 0, "");
+	if (pjob->ji_qs.ji_svrflags & JOB_SVFLG_SubJob &&
+			0 == subjob_index_to_offset(pjob->ji_parentaj,
+					get_index_from_jid(pjob->ji_qs.ji_jobid))) {
```

Comment:
I used to follow this style of coding to avoid accidental mistakes. But it will be much more natural to read if the number is on the right side of the operator and this is the standard followed throughout PBS code. I will change this in my next commit.